include/wx/charts/wxdoughnutandpiechartbase.h

```diff
@@ -40,6 +40,7 @@
 #include "wxchartslicedata.h"
 #include "wxdoughnutandpiechartoptionsbase.h"
 #include "wxchartarc.h"
+#include "wxchartobservers.h"
```

Comment:
This is not used by the base class so let's include it in the files where it is needed.